### ISSUE TITLE
[Accessibility] Does flutter support accessibility on Linux platform

### ISSUE THREAD

**Issue by XuWanchaoS** [flutter/flutter#91784]
## Use case
 I read the accessibility of flutter engine code, in shell/platform/linux, there are some files such as fl_accessibility_plugin.cc, it seems flutter support accessibility on linux platform now. But I test a flutter application, the accessibility doesn't work.

Is there anyone developing accessibility for linux platform?

**Comment by danagbemava-nc** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
